react-app/src/hooks/api/useBuildingsApi.ts

```diff
@@ -1,55 +1,104 @@
-const buildings1 = [
-  {
-    Id: 1,
-    ClassificationId: 0,
-    Classification: {
-      Name: 'Core operational',
-      Id: 0,
-    },
-    Address1: '2345 Example St.',
-    AgencyId: 1,
-    Agency: { Name: 'Smith & Weston' },
-    PID: 111333444,
-    IsSensitive: true,
-    UpdatedOn: new Date(),
-    Evaluations: [{ Value: 99888, Date: new Date() }],
-    Fiscals: [{ Value: 1235000, FiscalYear: 2024 }],
-  },
-  {
-    Id: 2,
-    ClassificationId: 5,
-    Classification: {
-      Name: 'Disposed',
-      Id: 5,
-    },
-    Address1: '6432 Nullabel Ln.',
-    AgencyId: 1,
-    Agency: { Name: 'Smith & Weston' },
-    PID: 676555444,
-    IsSensitive: false,
-    UpdatedOn: new Date(),
-    Evaluations: [{ Value: 999988, Date: new Date() }],
-    Fiscals: [{ Value: 1235000, FiscalYear: 2024 }],
-  },
-];
+import { IFetch } from '../useFetch';
+import { Agency } from './useAgencyApi';
+export interface Evaluation {
+  Date: Date;
+  Value: number;
+}
+export interface Fiscal {
+  FiscalYear: number;
+  EffectiveDate: Date;
+  Value: number;
+}
+export interface Classification {
+  Name: string;
+}
+export interface BuildingConstructionType {
+  Name: string;
+}
+export interface BuildingPredominateUse {
+  Name: string;
+}
+export interface BuildingOccupantType {
+  Name: string;
+}
+export interface PropertyType {
+  Name: string;
+}
 
-const useBuildingsApi = () => {
+export type GeoPoint = {
+  x: number;
+  y: number;
+};
+export interface Building {
```

Comment:
Since the parcels hooks got merged, you have access to many of these types/interfaces already. No reason to duplicate them.
There's also a Property interface that you could extend for the Building instead of having it stand on its own.